client/extensions/woocommerce/woocommerce-services/state/label-settings/selectors.js

```diff
@@ -34,6 +34,11 @@ export const getLabelSettingsStoreOptions = ( state, siteId = getSelectedSiteId(
 	return form && form.storeOptions;
 };
 
+export const getLabelSettingsUserMeta = ( state, siteId = getSelectedSiteId( state ) ) => {
+	const form = getLabelSettingsForm( state, siteId );
```

Comment:
Maybe use getLabelsSettingsFormMeta. It already gets meta from the form. Either that or check if meta exists. You risk of getting: `Cannot read property 'user' of undefinded`.

Reply:
Good catch!